packages/generator-helper/src/types.ts

```diff
@@ -130,3 +131,59 @@ export type GeneratorManifest = {
   version?: string
   requiresEngineVersion?: string
 }
+
+export type SqlQueryOutput = {
+  name: string
+  source: string
+  documentation: string | null
+  parameters: SqlQueryParameterOutput[]
+  resultColumns: SqlQueryColumnOutput[]
+}
+
+export type SqlQueryParameterOutput = {
+  name: string
+  query: string
+  // can refer to user-defined enums, so does not map to QueryIntrospectionType 1:1
+  typ: string
+  documentation: string | null
+}
+
+export type SqlQueryColumnOutput = {
+  name: string
+  // can refer to user-defined enums, so does not map to QueryIntrospectionType 1:1
+  typ: string
+}
+
+export type QueryIntrospectionType =
```

Comment:
It would be great if we could DRY this type with the existing `PrismaType` in `runtime/utils/deserializeRawResults.ts`